### PR TITLE
Add sass-lint to webpack

### PR DESCRIPTION
**Pull request by alexose**
Along with #2792, this adds sass linting into the webpack pipeline, allowing for it to run on rebuild.

There are a bunch of warnings, however, which we should fix.

### CODE REVIEW

File: config/webpack.config.js [hunk 1]
```diff
@@ -96,6 +102,11 @@ var configGenerator = (options) => {
               NODE_ENV: JSON.stringify(process.env.NODE_ENV)
           }
       }),
+      new sassLintPlugin({
+        configFile: 'config/.sass-lint.yml',
```
Comment (awong-dev) [inline config/webpack.config.js]: Don't use a dot-file here. Removing the leading dot keeps it easily visible in the filesystem.

Comment (awong-dev) [inline config/webpack.config.js]: Also, is there an existing sass-lint? I can't find it....

Comment (alexose) [inline config/webpack.config.js]: Erp, that was a mistake.  It should just be pointing at `config/sass-lint.yml` (which already exists in master).

**Comment by alexose** [comment]
Closing in favor of a better solution: https://github.com/department-of-veterans-affairs/vets-website/pull/2860

**Pull request closed without merging**